doc/reference_data

```diff
@@ -1 +1 @@
-Subproject commit c6f0638fb47810929048346f5359556c83ba448d
+Subproject commit 94a283186c7421cd25bfbb56774d986fce2c4c3e
```

Comment:
Not quite clear on why it changed as the visual output hasn't and the Elements example didn't require any change. But we'll delete this Tutorial very soon anyway and regenerate the reference_data.